tripy/nvtripy/frontend/ops/plugin.py

```diff
@@ -17,7 +17,10 @@
 
 from typing import List, Sequence, Tuple, Union
 
+import numpy as np
```

Comment:
Any way we can remove this NumPy dependency? Looks like we convert all NumPy array to lists anyways - could we just accept lists instead?

Reply:
No, unfortunately it does not seem possible. The `output_descs = trtp_op.register_func(*input_descs, attrs)` call below has to conform to the QDP API: the `attrs` must have a NumPy array for any non-scalar attribute. So even if we accepted Python lists through the Tripy API, we'd have to convert it to a NumPy array before the `trtp_op.register_func()`.

Reply:
Is it feasible to extend the TRT API to accept lists? That seems like a more natural representation than allowing NumPy arrays but constraining them to 1D.

Reply:
The 1D restriction is temporary -- eventually we plan to support N-D NumPy arrays. The reason to accept NumPy was so that at that point we can make a smooth transition by not having to change the API.